### ISSUE TITLE
Screen reader does not announce the state as selected for pagination links in pagination section in windows.

### ISSUE THREAD

**Issue by msftedad** [ng-bootstrap/ng-bootstrap#4893]
**Test Environment:**
OS: Windows 11 version 25H2 (OS Build 26220.7523)
Browser: Edge (Version 144.0.3719.35)
Screen Reader: NVDA (2025.3.2)
URL: https://angular-ui.github.io/bootstrap/#!#getting_started:~:text=Pagination%20(ui.bootstrap.pagination) 

**User Impact:**
Screen reader users cannot identify which pagination link is currently selected because its selected state is not announced. This makes it difficult to track their position, causing confusion and navigation errors when moving through pages.

**Pre-Requisites:**
Enable NVDA

**Repro Steps:**

1. Open URL: https://angular-ui.github.io/bootstrap/#!#getting_started:~:text=Pagination%20(ui.bootstrap.pagination) in edge browser in windows.
2. 'Angular directives for Bootstrap - Pagination' page will open.
3. Press tab key to navigate to the selected '4' pagination link.
4. Verify whether screen reader announces the state as selected for pagination links in pagination section in windows or not.

**Actual Result:**
Screen reader does not announce the state as selected for pagination links in pagination section in windows.
Screen reader announces: 4 link visited same page

**Refer Attachment:**

1. Screen reader does not announce the state as selected for pagination links in pagination section in windows.png
2. Screen reader does not announce the state as selected for pagination links in pagination section in windows.mp4

https://github.com/user-attachments/assets/9bb43d08-b505-4fb1-b4d7-2cb18fd06d01
<img width="969" height="521" alt="Image" src="https://github.com/user-attachments/assets/3126eadb-8c0d-49aa-8f4e-3ea5dd87d038" />

**Expected Result:**
Screen reader should announce the state as selected for pagination links in pagination section in windows.
Screen reader should announce: 4 link visited selected same page

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Issue is fixed. You can close this issue
